ext/opcache/Optimizer/sccp.c

```diff
@@ -844,13 +844,20 @@ static inline int ct_eval_func_call(
 				|| zend_string_equals_literal(name, "urldecode")
 				|| zend_string_equals_literal(name, "rawurlencode")
 				|| zend_string_equals_literal(name, "rawurldecode")
-				|| zend_string_equals_literal(name, "php_uname")) {
+				|| zend_string_equals_literal(name, "php_uname")
```

Comment:
Not even related to this PR because this is old (I misread the diff), but it looks as if 
1. `php_uname` is safely not emitting notices, and this can be kept in
2. https://www.php.net/manual/en/function.php-uname.php documents that it expects a single character. No notices or exceptions are emitted if the string is empty or has a length other than one, or has a character not in the documented supported values, or any other reason.

Maybe starting to emit E_DEPRECATED for any strings that aren't characters in that list (and after that, removing this (not here)) would be useful, but obviously out of scope. I didn't see related issues in https://bugs.php.net/search.php?search_for=php_uname&boolean=0&limit=30&order_by=&direction=DESC&cmd=display&status=All&bug_type=All&project=All&php_os=&phpver=&cve_id=&assign=&author_email=&bug_age=0&bug_updated=0&commented_by=